mlflow/transformers/llm_inference_utils.py

```diff
@@ -12,6 +21,167 @@
     "text-generation": [_LLM_INFERENCE_TASK_COMPLETIONS, _LLM_INFERENCE_TASK_CHAT],
 }
 
+_LLM_INFERENCE_COMPLETIONS_OUTPUT_SCHEMA = Schema(
```

Comment:
Is there any particular reason we use different schema as CHAT_MODEL_OUTPUT_SCHEMA? Given that the goal is to unify LLM input/output schema, we should eventually merge them into one.

Reply:
Agree that we should standardize on one, otherwise it's difficult for end-users to switch between different model backends when evaluating LLMs for their use-cases!

Reply:
Another note—would it be possible to define `COMPLETIONS_MODEL_OUTPUT_SCHEMA` in `mlflow/types/llm.py` instead? I think this can be helpful for people building completion models using custom pyfuncs as well, and will make it easier in the future if we want to define `mlflow.pyfunc.CompletionsModel`

Reply:
Makes sense, I'll update them to use `CHAT_MODEL_OUTPUT_SCHEMA` and `COMPLETIONS_MODEL_OUTPUT_SCHEMA` (newly defined) instead.

Reply:
Awesome, thanks!

Reply:
Updated the signatures used by the inference tasks, and added the missing fields in the output dict. PTAL, thanks!